### ISSUE TITLE
How to train with multiple GPUs?

### ISSUE THREAD

**Issue by lathampratti** [idealo/image-super-resolution#55]
Hello,
What modifications do I need to distribute the training across multiple GPUs?

Thanks for the help.

**Comment by Flyzzz** [comment]
have you slove this problom? train on gpu

**Comment by lathampratti** [comment]
No, I am still looking for a solution to train this with multiple GPUs.
